.github/workflows/test.yml

```
@@ -7,21 +7,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        terraform_version: [ "1.0.x" ]
+        terraform_version: [ "1.2.x" ]
```

I would prefer to add all three latest supported versions so:
```suggestion
        terraform_version: [ "1.0.x", "1.1.x", "1.2.x" ]